### ISSUE TITLE
Accessibility: unreadable element

### ISSUE THREAD

**Issue by jansche** [bpetetot/conference-hall-beta#855]
**What is the issue?**
![image](https://user-images.githubusercontent.com/22521588/100219392-3c286b80-2f16-11eb-9d86-0ca449262ac1.png)
The element shown above is unaccessible for screen readers. It never gets focus.

**What is the expected behavior?**
The tooltip/hint windows provides essential information and should be navigate-able/readable for screen readers.

**What is your environment?**
JAWS screen reader

- OS: Windows and Mac
- Browser: Edge

**Comment by bpetetot** [comment]
Thanks again for the feedback. ;)
To avoid the tooltip accessibility issue, we decide to move the hint as a description (always visible)

![image](https://user-images.githubusercontent.com/516360/101295651-2a509d80-381f-11eb-88f5-a803f488e5ef.png)

Fix: 223de35f3b5416c9bc68446407b71c145b7c9be8
